server/src/task/list/list.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { ListPermission } from '@prisma/client'
+import { IsEnum, IsNotEmpty, IsOptional, IsString } from 'class-validator'
+import { enumInvalidMessage } from '../../utilities/validation'
+
+export class CreateTasklistDto {
+    @IsString()
+    @IsNotEmpty()
+    name: string
```

Comment:
I think we don't need the `@IsString()` here if we already have the `@IsNotEmpty()`